helios-tools/src/main/java/com/spotify/helios/cli/Utils.java

```diff
@@ -54,7 +54,7 @@
   }
 
   public static HeliosClient getClient(final Target target, final PrintStream err,
-                                       final String username) {
```

Comment:
allows the CLI options to be passed down to this method for use in constructing the HeliosClient

kind of ugly but I didn't want to add `hostnameVerificationEnabled` as a field to the existing `Target` parameter as it didn't really seem to fit there.